tests/integration/integrationtests.py

```diff
@@ -5,53 +5,78 @@
 
 import utils
 
+VALID_RATES = [
+    'Staff Undergraduate',       # STUG
+    'Staff Graduate',            # STGR
+    'OSU Staff Dependent UG',    # SDUG
```

Comment:
What do these 4 character codes mean in the comments?

Reply:
They are the student rate codes from STVRATE which we should only expect.